demos/browser/app/meetingV2/styleV2.scss

```diff
@@ -872,4 +872,48 @@ a.markdown:active {
 
 .card-header {
   font-size: 140%;
+}
+
+#flow-disabled-permissions {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 2rem;
+
+  .disabled-title {
+    margin-bottom: 3rem;
+    color: $gray-900;
+  }
+
+  .disabled-desc-container {
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    max-width: 880px;
+    padding: 80px;
+    border: 2px solid $green;
+    border-radius: 4px;
```

Comment:
Future: I think the width sizes for most of these classes should be in `rem`, not `px`. Again, good enough for now.